lib/agent0/agent0/hyperdrive/crash_report/crash_report.py

```diff
@@ -97,24 +100,42 @@ def log_hyperdrive_crash_report(trade_result: TradeResult) -> None:
     wallet_dict = _hyperdrive_wallet_to_dict(trade_result.agent.wallet)
     formatted_agent_wallet = json.dumps(wallet_dict, indent=4, cls=ExtendedJSONEncoder)
 
-    # TODO Once pool_info and pool_config are objects, we need to add a conversion function
-    # to convert to dict
-    formatted_pool_info = json.dumps(trade_result.pool_info, indent=4, cls=ExtendedJSONEncoder)
-    formatted_pool_config = json.dumps(trade_result.pool_config, indent=4, cls=ExtendedJSONEncoder)
-
     formatted_agent_info = _hyperdrive_agent_to_dict(trade_result.agent)
     formatted_agent_info = json.dumps(formatted_agent_info, indent=4, cls=ExtendedJSONEncoder)
 
+    # TODO Once pool_info and pool_config are objects, we need to add a conversion function to convert to dict
+    formatted_pool_config = json.dumps(trade_result.pool_config, indent=4, cls=ExtendedJSONEncoder)
+    formatted_pool_info = json.dumps(trade_result.pool_info, indent=4, cls=ExtendedJSONEncoder)
+
+    # additional info we put together into a nice dict
+    additional_info = {
```

Comment:
Nit, this information should be gathered with the rest of the pool config and pool info with trade_result (https://github.com/delvtech/elf-simulations/blob/46ee1d7dbd24ca970243706f6fde5be42ed867b5/lib/agent0/agent0/hyperdrive/exec/execute_agent_trades.py#L91). The idea here is that we grab all this information ASAP during crash, not at the moment of crash reporting handling.

Reply:
tried this in 23ad2dbfab8c28e59a2f0841fa24912c7ca644ef